projects/re2c.org/package.yml

```diff
@@ -0,0 +1,32 @@
+distributable:
+  url: https://github.com/skvadrik/re2c/archive/refs/tags/{{ version.raw }}.tar.gz
```

Comment:
use the official release tarballs: https://github.com/skvadrik/re2c/releases/download/3.0/re2c-3.0.tar.xz we prefer this since this is what the maintainer intends for us to use to build from source. We only use the autogenerated github sources are only used if necessary.

this also means you can drop autoconf/make and autogen.sh